README.md

```diff
@@ -78,9 +88,80 @@ Check using the following command:
 oc get pods -n cluster-reg-config
 ```
 
+# Onboard a hub cluster
+
+## hub cluster pre-req
+- The managed hub must be MCE 2.0.0+
+- On the managed hub, the multiclusterengine CR must have the managed-service-account enabled.
+`oc edit multiclusterengine`
+then set the following
+```
+    - enabled: true
+      name: managed-service-account
+```
+## Onboarding
+
+1. Create config secret on the external cluster to access the hub cluster:
+
+```bash
+echo '
+apiVersion: v1
+data:
+  kubeConfig: <base64 kubeconfig toward the hub>
```

Comment:
We need to sync up with what `Local Development` section in README has which contradicts this
```oc apply -f hack/hubconfig.yaml
oc create secret generic mce-kubeconfig-secret --from-file kubeConfig # Expects a kubeconfig file named kubeConfig
```

Reply:
I guess we have to add the `-n`